receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -147,6 +147,18 @@ metrics:
       if_enabled: This metric will be disabled in v0.82.0 and removed in v0.85.0.
     gauge:
       value_type: double
+  container.cpu.limit:
+    enabled: false
+    description: "CPU limit set for the container. This metric is only reported if the container has limits set with -cpus, -cpuset-cpus or -cpu-quota."
```

Comment:
I'm not sure it's necessary to include `This metric is only reported if the container has limits set with -cpus, -cpuset-cpus or -cpu-quota.` in the reported telemetry considering it must have been set to be reported. Maybe a good candidate for the `if_enabled` warning?

Reply:
I'm not sure, as the warning message will be:

```
[WARNING] `container.cpu.limit` should not be enabled: This metric is only reported if the container has limits set with -cpus, -cpuset-cpus or -cpu-quota.
```
does it make sense using the extended documentation field instead?

Reply:
I took the `extended_documentation approach here: [9b8ec2a](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/commits/9b8ec2a97cf6ba01e862cf65064f6069da0745a3), but we can change it if you think a warning is better.